app/views/admin/organizations/index.html.erb

```diff
@@ -32,10 +42,16 @@
         </div>
       </div>
       <div class="col-2 text-right">
-        <% if policy([:manage, organization]).destroy? %>
+        <% if Flipper.enabled?(:new_permissions_system_aug_2020, current_admin) %>
```

Comment:
There might be a bug here. Doesn't this bit mean once we enable the flipper flag, everyone will see a link?

Reply:
_ditto_